.github/workflows/run_tests.yml

```diff
@@ -28,7 +28,7 @@ jobs:
       fail-fast: false
       matrix:
         os: ["ubuntu-latest"]
-        python: ["3.9", "3.10", "3.11", "3.12"]
+        python: ["3.10", "3.11", "3.12"]
```

Comment:
Try to run 3.13, too. 3.13.0 has already been released. Some of our dependencies might be broken, but let's give it a try.